docs/examples/custom_operations/numba_function.ipynb

```diff
@@ -0,0 +1,468 @@
+{
```

Comment:
<p> and a list of input shapes<s>, which contains the input shape</s></p><p> </p><blockquote>The first dimension is the index of the output/input</blockquote><p> </p><p>It's not a multidimensional array, but a nested list. I'd rephrase it:</p><p> </p><p>The outer index applied to outs/ins is the index of the respective output/input; the inner index indicates the sample in the batch.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>